### PR TITLE
Revert PR 2681

### PR DESCRIPTION
**Pull request by aerorahul**
# Description
This PR:
- reverts #2681 in part 
- keeps some changes for `RUN`.
- is a hotfix
- should be merged ASAP after consensus w/ @guillaumevernieres @CatherineThomas-NOAA @WalterKolczynski-NOAA 

# Type of change
- Bug fix (fixes something broken)

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: parm/config/gfs/config.ufs [hunk 1]
```diff
@@ -414,6 +413,11 @@ if [[ "${skip_mom6}" == "false" ]]; then
       CHLCLIM="seawifs_1998-2006_smoothed_2X.nc"
       MOM6_RESTART_SETTING='r'
       MOM6_RIVER_RUNOFF='False'
+      if [[ ${RUN} == "gfs" || "${RUN}" == "gefs" ]]; then
+        MOM6_DIAG_MISVAL="-1e34"
+      else
+        MOM6_DIAG_MISVAL="0"
```
Comment (aerorahul) [inline parm/config/gfs/config.ufs]:
```suggestion
        MOM6_DIAG_MISVAL="0.0"
```

File: parm/config/gfs/config.ufs [hunk 2]
```diff
@@ -434,8 +438,10 @@ if [[ "${skip_mom6}" == "false" ]]; then
       TOPOEDITS="ufs.topo_edits_011818.nc"
       if [[ ${RUN} == "gfs" || "${RUN}" == "gefs" ]]; then
         MOM6_DIAG_COORD_DEF_Z_FILE="interpolate_zgrid_40L.nc"
+        MOM6_DIAG_MISVAL="-1e34"
       else
         MOM6_DIAG_COORD_DEF_Z_FILE="oceanda_zgrid_75L.nc"
+	MOM6_DIAG_MISVAL="0"
```
Comment (aerorahul) [inline parm/config/gfs/config.ufs]:
```suggestion
	MOM6_DIAG_MISVAL="0.0"
```

File: parm/config/gfs/config.ufs [hunk 3]
```diff
@@ -453,8 +459,10 @@ if [[ "${skip_mom6}" == "false" ]]; then
       eps_imesh="1.0e-1"
       if [[ ${RUN} == "gfs" || "${RUN}" == "gefs" ]]; then
         MOM6_DIAG_COORD_DEF_Z_FILE="interpolate_zgrid_40L.nc"
+        MOM6_DIAG_MISVAL="-1e34"
       else
         MOM6_DIAG_COORD_DEF_Z_FILE="oceanda_zgrid_75L.nc"
+        MOM6_DIAG_MISVAL="0"
```
Comment (aerorahul) [inline parm/config/gfs/config.ufs]:
```suggestion
        MOM6_DIAG_MISVAL="0.0"
```

File: parm/config/gfs/config.ufs [hunk 4]
```diff
@@ -473,8 +481,10 @@ if [[ "${skip_mom6}" == "false" ]]; then
...
```
Comment (aerorahul) [inline parm/config/gfs/config.ufs]:
```suggestion
        MOM6_DIAG_MISVAL="0.0"
```

**Review by aerorahul — COMMENTED** [review]
don't take chances w/ zero.zero

**Review by WalterKolczynski-NOAA — DISMISSED** [review]

**Comment by JessicaMeixner-NOAA** [comment]
FYI - My C48mx500 WCDA test succeeded with this branch: /scratch1/NCEPDEV/climate/Jessica.Meixner/tryagainocnpost/testciC48mx500wcda/ 

C384 test is not far enough along to check yet

**Comment by aerorahul** [comment]
I don't know what GH is checking for here
Screenshot 2024-06-28 at 2 38 32 PM

**Comment by RussTreadon-NOAA** [comment]
I copied the `config.ufs` from `JessicaMeixner-NOAA:feature/updatefillvalues` at ff74f8e6 to Hera `/scratch1/NCEPDEV/stmp2/Russ.Treadon/EXPDIR/pr2700_wcda`.  The 20210324 18Z gdasfcst was rewound and rebooted.  The model failed in the same way it did before.   

Do we need to back up the 20210324 12Z start of C48mx500_3DVarAOWCDA and rerun from the first half-cycle gdasfcst?

File: parm/config/gfs/config.ufs [hunk 5]
```diff
@@ -434,8 +438,10 @@ if [[ "${skip_mom6}" == "false" ]]; then
       TOPOEDITS="ufs.topo_edits_011818.nc"
       if [[ ${RUN} == "gfs" || "${RUN}" == "gefs" ]]; then
         MOM6_DIAG_COORD_DEF_Z_FILE="interpolate_zgrid_40L.nc"
+        MOM6_DIAG_MISVAL="-1e34"
       else
         MOM6_DIAG_COORD_DEF_Z_FILE="oceanda_zgrid_75L.nc"
+	MOM6_DIAG_MISVAL="0.0"
```
Comment (DavidHuber-NOAA) [inline parm/config/gfs/config.ufs]:
```suggestion
        MOM6_DIAG_MISVAL="0.0"
```

**Comment by JessicaMeixner-NOAA** [comment]
> I copied the `config.ufs` from `JessicaMeixner-NOAA:feature/updatefillvalues` at [ff74f8e](https://github.com/NOAA-EMC/global-workflow/commit/ff74f8e65617c9dec63f095588f568723d8017e7) to Hera `/scratch1/NCEPDEV/stmp2/Russ.Treadon/EXPDIR/pr2700_wcda`. The 20210324 18Z gdasfcst was rewound and rebooted. The model failed in the same way it did before.
> 
> Do we need to back up the 20210324 12Z start of C48mx500_3DVarAOWCDA and rerun from the first half-cycle gdasfcst?

@guillaumevernieres  - can you confirm, but   I'm thinking you will because this will effect output from the first gdas forecast half cycle.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Comment by AndrewEichmann-NOAA** [comment]
> > I copied the `config.ufs` from `JessicaMeixner-NOAA:feature/updatefillvalues` at [ff74f8e](https://github.com/NOAA-EMC/global-workflow/commit/ff74f8e65617c9dec63f095588f568723d8017e7) to Hera `/scratch1/NCEPDEV/stmp2/Russ.Treadon/EXPDIR/pr2700_wcda`. The 20210324 18Z gdasfcst was rewound and rebooted. The model failed in the same way it did before.
> > Do we need to back up the 20210324 12Z start of C48mx500_3DVarAOWCDA and rerun from the first half-cycle gdasfcst?
> 
> @guillaumevernieres - can you confirm, but I'm thinking you will because this will effect output from the first gdas forecast half cycle.

We ran the `WCDA-3DVAR-C48mx500` test with this reversion and got physical number back in the increments.

**Comment by guillaumevernieres** [comment]
> > I copied the `config.ufs` from `JessicaMeixner-NOAA:feature/updatefillvalues` at [ff74f8e](https://github.com/NOAA-EMC/global-workflow/commit/ff74f8e65617c9dec63f095588f568723d8017e7) to Hera `/scratch1/NCEPDEV/stmp2/Russ.Treadon/EXPDIR/pr2700_wcda`. The 20210324 18Z gdasfcst was rewound and rebooted. The model failed in the same way it did before.
> > Do we need to back up the 20210324 12Z start of C48mx500_3DVarAOWCDA and rerun from the first half-cycle gdasfcst?
> 
> @guillaumevernieres - can you confirm, but I'm thinking you will because this will effect output from the first gdas forecast half cycle.

@RussTreadon-NOAA , you need to reboot the entire test, starting from the first 1/2 cycle.

**Comment by RussTreadon-NOAA** [comment]
Thank you @guillaumevernieres .  I've done so and C48mx500_3DVarAOWCDA is almost back up to the 18Z gdasfcst.

**Pull request merged**